### PR TITLE
Update javascript.json

### PR DESCRIPTION
**Pull request by eugenelim**
Added RxJS.Live conference

### CODE REVIEW

File: conferences/2019/javascript.json [hunk 1]
```diff
@@ -1055,6 +1055,16 @@
     "cfpUrl": "https://2019.jsconfkorea.com/en/call-for-proposals",
     "cfpEndDate": "2019-04-26"
   },
+  {
+    "name": "RxJS Live",
+    "url": "https://www.rxjs.live/",
+    "startDate": "2019-09-05",
+    "endDate": "2019-09-06",
+    "city": "Las Vegas, CA",
+    "country": "USA",
```
Comment (nimzco) [inline conferences/2019/javascript.json]: Tests are failing because this should be `U.S.A.` for consistency. :)

Comment (eugenelim) [reply conferences/2019/javascript.json]: added the fix

Comment (nimzco) [reply conferences/2019/javascript.json]: 👌

**Comment by nimzco** [comment]
Thanks!

File: conferences/2019/javascript.json (hunk 1 above)
Comment (eugenelim) [reply conferences/2019/javascript.json]: thanks!

**Pull request merged**